nat-lab/tests/utils/connection_util.py

```diff
@@ -323,6 +329,26 @@ def generate_connection_tracker_config(
 ) -> List[ConnectionTrackerConfig]:
     lan_addr = LAN_ADDR_MAP[connection_tag]
     ctc_list = [
+        ConnectionTrackerConfig(
+            "nlx_1",
+            nlx_1_limits,
+            FiveTuple(
+                protocol="udp",
+                src_ip=lan_addr,
+                dst_ip=str(config.NLX_SERVER.get("ipv4")),
+                dst_port=51820,
```

Comment:
Is it really "five"?

Reply:
:question:

Reply:
It's 4 elements, but it was modelled after the VPN setup which uses the same FiveTuple for 4 elements. Didn't want to touch that yet

Reply:
OK, then